# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/binast/lazy/debug/Frame-01.binjs
# Commit: 73c33bf110d6
# Full Hash: 73c33bf110d6034266cbe615b9e018476a128d63
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 2
# Description:
#   Bug 1600204 - Part 2: Remove Debugger.Frame.prototype.generator since it is already on script. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55232
# ==============================================================================

diff -r d70ec1e79887 -r 73c33bf110d6 js/src/jit-test/tests/binast/lazy/debug/Frame-01.binjs
Binary file js/src/jit-test/tests/binast/lazy/debug/Frame-01.binjs has changed